projects/packages/explat/src/class-explat.php

```diff
@@ -7,10 +7,38 @@
 
 namespace Automattic\Jetpack;
 
+use Automattic\Jetpack\Connection\Rest_Authentication;
+use Automattic\Jetpack\Explat\REST_Controller;
+
 /**
  * Class description.
  */
 class Explat {
```

Comment:
What do you think about changing this to `ExPlat` to match the same capitalization of the [ExPlatClient](https://github.com/Automattic/wp-calypso/blob/9eed9101ec1468cbce42d4c5e55bbf819134c270/packages/explat-client/src/create-explat-client.ts#L71)?